WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -188,26 +188,30 @@ class PostCoordinator: NSObject {
             }
 
             // Ensure that all synced media references are up to date
-            post.media.forEach { media in
```

Comment:
Please be cautious about making changes to `PostCoordinator` because the large potion of it is or will be rewritten. `prepareToSave` is unchanged, so it's a safe change.

Reply:
Thanks @kean for the call out 🙇 ! This PR mainly focuses on the logic related to processing blocks before saving a post, which requires touching up some bits of the `PostCoordinator`. I'm planning to modify only necessary here, so hopefully there won't be much conflict with ongoing changes. In any case, I'll be happy to contribute to the new version by applying the same updates.